client/components/cards/attachments.js

```diff
@@ -1,5 +1,12 @@
 Template.attachmentsGalery.events({
-  'click .js-add-attachment': Popup.open('cardAttachments'),
+  'click .js-add-attachment'() {
+    Popup.open('cardAttachments').apply(this, arguments);
+
+    $('html').pasteImageReader(function(results) {
+      Popup.pasted = results;
```

Comment:
The paste can only be fired by keyboard Ctrl-V. Here we temporarily store the content in Popup.pasted, later we get from it then upload.